components/brave_wallet/common/btc_like_serializer_stream.h

```diff
@@ -9,13 +9,13 @@
 #include <vector>
 
 #include "base/containers/span.h"
-#include "base/memory/raw_ptr.h"
 
 namespace brave_wallet {
 
 class BtcLikeSerializerStream {
  public:
-  explicit BtcLikeSerializerStream(std::vector<uint8_t>* to) : to_(to) {}
+  BtcLikeSerializerStream();
+  ~BtcLikeSerializerStream();
 
   void Push8(uint8_t i);
   void Push16(uint16_t i);
```

Comment:
As we are touching this will it also makes sense to convert these methods `PushX(uintX_t i);` to `PushX(base::StrictNumeric<uintX_t> i);`?

Reply:
After discussing things a bit with core, I'm not holding my breath we can make `StrictNumeric` more ergonomic. I have added `StrictNumeric` to those push interfaces.